src/utils/voice_recorder.py

```
@@ -0,0 +1,187 @@
+
+            # 斷開連接
+            await self.voice_client.disconnect()
+            self.is_connected = False
+            self.voice_client = None
+            self.start_time = None
+            self._disconnect_task = None
+
+            logger.info("已斷開語音頻道連接")
+            return True
+
+        except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)